src/i18n/en-US/modelPlan/modelToOperations.ts

```diff
@@ -660,11 +660,9 @@ export const modelToOperationsMisc: Record<string, any> = {
     changeAnswer: 'Want to change your answer?',
     goToQuestion: 'Go to the question',
     added: 'Added',
-    noSuggestedHeading:
-      'There are no suggested milestones that match your search.',
+    noSuggestedHeading: 'There are no suggested milestones for your model.',
     noSuggestedDescription:
-      'MINT hasn’t suggested any milestones for your model that match your search. You may also try checking <link1>all available milestones</link1>. As you and your team fill out more of your <link2>Model Plan</link2>, MINT may display suggested common milestones that match your search. If you’re searching for a milestone that you believe should be a part of MINT, please contact the MINT Team at <email1>MINTTeam@cms.hhs.gov</email1>.',
-    dontSeeMilestone: 'Don’t see a milestone you need?',
+      'MINT hasn’t yet suggested any milestones for your model. As you and your team fill out more of your Model Plan, MINT may display suggested common milestones for your model here. To view all available common milestones, click the button above labeled “All common milestones”.',
```

Comment:
There should be a link around `Model Plan` here.

<img width="311" alt="Screenshot 2025-05-16 at 7 40 37 AM" src="https://github.com/user-attachments/assets/66cfc992-7056-4156-97a9-abfc75d86ca8" />

To embed links within the translation you can do something like

```
As you and your team fill out more of your <link1>Model Plan</link1>, MINT may
```

(link is a reserved tag, so we generally use link1, link2, etc)

Then within the code, you interpolate that using the `<Trans />` component like so 

```
<Trans
    i18nKey="milestoneLibrary:noSuggestedDescription"
    components={{
      link1: (
        <UswdsReactLink
          to={`/models/${modelID}/collaboration-area`}
        > </UswdsReactLink>
      )
    }}
  />
```